lib/internal/errors.js

```diff
@@ -119,11 +119,9 @@ const prepareStackTrace = (globalThis, error, trace) => {
   // Error: Message
   //     at function (file)
   //     at file
-  let errorString;
+  let errorString = ErrorPrototypeToString(error);
   if (kIsNodeError in error) {
     errorString = `${error.name} [${error.code}]: ${error.message}`;
-  } else {
-    errorString = ErrorPrototypeToString(error);
```

Comment:
Hey, thanks for the PR. Can you explain why you want to refactor this? It seems to me that we wouldn't want to call `ErrorPrototypeToString` if we don't have to.

Reply:
Hi @aduh95 , does calling it count as an expensive process? I just wanted to set it as a default value to the variable so it makes the code more simpler, but if the process is expensive (takes a lot of memory or time), there is no need to my changes.

Reply:
It's non-zero, we would have to run benchmarks to know how bad it is (maybe not that much), but since this code is rather critical for performance across all APIs, I'm tempted to take a precocious approach and I would need more reasons than code readability (which is subjective, not everyone will agree your version is more readable) to approve this change.

Reply:
You're right, my changes are about code readability only, so we can close this pr.